src/joAbstractLinearOperator/extra_functions.jl

```diff
@@ -20,6 +20,83 @@ iswide(A::joAbstractLinearOperator{DDT,RDT}) where {DDT,RDT} = (A.m < A.n)
 # iscomplex(jo)
 iscomplex(A::joAbstractLinearOperator{DDT,RDT}) where {DDT,RDT} = !(DDT<:Real && RDT<:Real)
 
+# iscomplex(jo)
+iscomplex_b(A::joAbstractLinearOperator{DDT,RDT}) where {DDT,RDT} = !(DDT<:Real && RDT<:Real)
+
+
+"""
+    normest(jo;numiters, tol)
+Approximate spectral norm (largest singular value) through power iterations. For small matrices a single iteration 
+should be enough. 
+"""
+
+function normest(A::joAbstractLinearOperator{DDT,RDT};numiters::Integer=5,tol::Float64=0.,verbose::Bool=false) where {DDT,RDT}   
+
+    spec_norm_aprox = 0
+
+    u_hat = DDT<:Real ? jo_convert(DDT,randn(A.m)) : jo_convert(DDT,complex.(randn(A.m),randn(A.m)))
+    u_hat  = u_hat / norm(u_hat,2)
+    v_hat  = deepcopy(u_hat)
+    for i=1:numiters
+        result_u = A'*u_hat
+        v_hat    = result_u / norm(result_u,2)
+      
+        result_v = A*v_hat
+        u_hat    = result_v / norm(result_v,2)
+
+        spec_norm_aprox = u_hat'*(A*v_hat)
+    end
+    return spec_norm_aprox
+end
+
+"""
+    isposdef(jo; semi=false)
+
+approximate check that the operator is positive definite or positive semidefinite. 
+"""
+
+function isposdef(A::joAbstractLinearOperator{DDT,RDT},samples::Integer=3;tol::Float64=0.,semi::Bool=false,verbose::Bool=false) where {DDT,RDT}   
+    if !(issquare(A)) 
+        println("Matrix needs to be square for PD/PSD test")
```

Comment:
no "test"

Reply:
What do you mean no test? That this string should print out test? Or that we should return the Test and TEST variables as in the other scenarios?

Reply:
I mean the sentence is a bit weird.

Reply:
I see